src/components/markdown-render/h4.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const H4 = (props) => {
+  return (
+    <h4 className="docs-b-h4 sprk-b-TypeDisplayFour sprk-u-mbm sprk-u-Measure" {...props} />
```

Comment:
We talked about how this works, but it's a lil suspicious that this is a self closing h4.

Just documenting that we learned that in JSX "You can also use the self-closing form of the tag if there are no children"

Children is the main key word...h4s have children. But this seems to also works with other headline components so 🤷🏽‍♂️

Reply:
more info: https://reactjs.org/docs/jsx-in-depth.html